DDDEastAnglia/Areas/Admin/Controllers/SponsorController.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Drawing;
+using System.Drawing.Imaging;
+using System.IO;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using DDDEastAnglia.DataAccess;
+using DDDEastAnglia.DataAccess.SimpleData.Models;
+using DDDEastAnglia.Helpers;
+using SponsorModel = DDDEastAnglia.Areas.Admin.Models.SponsorModel;
+
+namespace DDDEastAnglia.Areas.Admin.Controllers
+{
+    public class SponsorController : Controller
+    {
+        private readonly ISponsorRepository sponsorRepository;
+        private readonly DefaultSponsorSorter _defaultSponsorSorter;
+
+        public SponsorController(ISponsorRepository sponsorRepository, DefaultSponsorSorter _defaultSponsorSorter)
```

Comment:
I had a discussion with @adrianbanks about this.. 

IMHO the `DefaultSponsorSorter` doesn't need an interface.. it's just behaviour.. in fact it could be a pure function.. as it takes an `IEnumerable<T>` and returns an `IEnumerable<T>`.. and doesn't have any dependencie so passing in a delegate would allow it to be changed?